config/manager/kustomization.yaml

```diff
@@ -1,9 +0,0 @@
-resources:
```

Comment:
accidental remove? I think we need this file.

Reply:
In this closed pr of mine, #1817, I was told to exclude these two files from my pr so I deleted them in this pr. Should i add them?

Reply:
Just don't include changes on this file in your PR.